docker-rancher/nodes.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file is not needed. Only the main azuredeploy and azuredeploy.parameters should be able to run the entire template